datasets.py

```diff
@@ -139,13 +140,18 @@ def __init__(self, transform):
         # self.scene_mapping = setup_scenemapping(self, '[name of dataset]')
         self.scene_mapping = NoneDict()
 
+        #Note: Any of the 'optional' attributes may be necessary depending on analysis and metrics, check before not filling in
+
+
         # Maps each label to number of supercategory group, which is listed in keys of GROUPINGS_TO_NAMES (optional)
         self.group_mapping = None
 
         # Labels that correspond to people (optional)
+        #Note: This refers to labels that describe people in images
```

Comment:
combine with the comment right above in line 149. maybe something like "Labels, that are entries from self.categories, which correspond to a person]" or along those lines